src/parsec/CMakeLists.txt

```diff
@@ -0,0 +1,367 @@
+##
```

Comment:
Does this work?

Reply:
I didn't try any of the CMake stuff that @tribad put in, but I didn't see any harm in including it for now...